monitorrent/plugin_managers.py

```diff
@@ -138,14 +140,14 @@ def get_watching_topics(self):
         return watching_topics
 
     def execute(self, engine):
-        for name, tracker in self.trackers.iteritems():
+        for name, tracker in self.trackers.items():
```

Comment:
That's how it works:
In Python 2, the methods items(), keys() and values() used to "take a snapshot" of the dictionary contents and return it as a list. It meant that if the dictionary changed while you were iterating over the list, the contents in the list would not change.

In Python 3, these methods return a view object whose contents change dynamically as the dictionary changes. Therefore, in order for the behavior of iterations over the result of these methods to remain consistent with previous versions, an additional call to list() has to be performed in Python 3 to "take a snapshot" of the view object contents.

TL; DR: We can remove list everywhere, assuming this will not break the functionality, however it's safer with list(), cause it keeps the behavior the same.